env/WCOSS_C.env

```diff
@@ -166,6 +166,11 @@ elif [ $step = "vrfy" ]; then
     export IOBUF_PARAMS="*:size=32M:count=4:verbose"
     export APRUNTRACK="$launcher -j1 -n1 -N1 -d1 -cc depth"
 
+elif [ $step = "metp" ]; then
+
+    export IOBUF_PARAMS="*:size=32M:count=4:verbose"
+    export APRUNTRACK="$launcher -j1 -n1 -N1 -d1 -cc depth"
+
```

Comment:
Double checking the metp block isn't needed in the other env files. I only see this added to the WCOSS_C env file.

Reply:
It shouldn't be. It has been working fine in GFSv16, and this env file was the only one with a vrfy block so just to be similar to vrfy I added the metp block.